rmf_schedule_visualizer/src/main.cpp

```diff
@@ -97,13 +891,25 @@ int main(int argc, char* argv[])
   }
 
   RCLCPP_INFO(
-    visualizer_data_node->get_logger(),
+    schedule_data_node->get_logger(),
     "Websocket server started on port: " + std::to_string(port));
+  
+  auto rviz_node = std::make_shared<ScheduleMarkerPublisher>(
+    "rmf_schedule_visualizer_marker_publisher",
+    schedule_data_node,
+    std::move(map_name),
+    rate);
+
+  rclcpp::executors::MultiThreadedExecutor executor{
+    rclcpp::executor::ExecutorArgs(), 3
```

Comment:
Why 3?

Reply:
Fixed. This should be 2. Thanks for pointing it out.